tests/model/test_model.py

```diff
@@ -1205,13 +1205,60 @@ def test_update_star_status(self, mocker, model, event_op,
         changed_ids = set(indexed_ids) & set(event_message_ids)
         for changed_id in changed_ids:
             assert model.index['messages'][changed_id]['flags'] == flags_after
-        (model.update_rendered_view.
-         has_calls([mocker.call(changed_id) for changed_id in changed_ids]))
+            assert model.update_rendered_view.has_calls(
+                    [mocker.call(changed_id)])
 
         for unchanged_id in (set(indexed_ids) - set(event_message_ids)):
             assert (model.index['messages'][unchanged_id]['flags'] ==
                     flags_before)
```

Comment:
Given the change to the method to add the `set_count`, we should ensure it is not called in this test?

Reply:
Added